src/components/Profile/ProfileHeading.tsx

```diff
@@ -1,23 +1,42 @@
-import React, { Fragment } from 'react'
+import classNames from 'classnames'
+import React from 'react'
+import { Flex, Text } from 'rebass/styled-components'
 
-import ProfileStyledLink from './ProfileStyledLink'
+import ProfileStyledLink from 'components/Profile/ProfileStyledLink'
 
-const WHITMAN_URL = 'http://jstu.art/ojNe'
+const WHITMAN_URL = 'https://jstu.art/ojNe'
+
+const ProfileHeadingText = ({ children, className, ...rest }: any) => (
+  <Text
+    as="span"
+    className={classNames('lh-title', className)}
+    color="textLight"
+    flex="1"
+    flexWrap="wrap"
+    fontFamily="serif"
+    fontSize={5}
+    fontWeight="body"
+    justifyContent={['center', 'flex-end']}
+    sx={{ display: 'inline-flex' }}
+    {...rest}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)